x/staking/keeper/val_state_change.go

```diff
@@ -107,6 +107,7 @@ func (k Keeper) BlockValidatorUpdates(ctx sdk.Context) []abci.ValidatorUpdate {
 // CONTRACT: Only validators with non-zero power or zero-power that were bonded
 // at the previous block height or were removed from the validator set entirely
 // are returned to Tendermint.
+// TODO(csongor): review this
 func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []abci.ValidatorUpdate, err error) {
```

Comment:
The changes here and in other places line up with the commit that Csongor made on the previous version. Good job!

https://github.com/wormhole-foundation/cosmos-sdk/commit/a4a502a1825534be997330f1420ac6fbf5597174#diff-94e0c747eae9963b85eb0c98a80be121d6a6d14740106842f238b3a33d584175R120